firecracker/microvm.py

```diff
@@ -262,6 +267,34 @@ def enable_rootfs(self, path_on_host: str) -> str:
             rootfs_filename = Path(path_on_host).name
             jailer_path_on_host = f"/opt/{rootfs_filename}"
             os.link(path_on_host, f"{self.jailer_path}/{jailer_path_on_host}")
+            return Path(jailer_path_on_host)
+        else:
+            return path_on_host
+
+    def mount_rootfs(self, path_on_host: Path) -> Path:
+        """Mount a rootfs to the VM.
+        """
+        self.mounted_fs = path_on_host
+        if self.use_jailer:
+            rootfs_filename = Path(path_on_host).name
+            jailer_path_on_host = Path(f"{DEVICE_BASE_DIRECTORY}/{rootfs_filename}")
+            if not (self.jailer_path / jailer_path_on_host).is_block_device():
+                device_vm_path = f"{DEVICE_BASE_DIRECTORY}/{self.vm_id}"
+                subprocess.run(["mkdir", "-p", device_vm_path],
+                               check=True,
+                               capture_output=True)
+                path_to_mount = f"{device_vm_path}/{rootfs_filename}"
+                os.symlink(path_on_host, path_to_mount)
+                os.system(f"cp -vap /dev/dm* {self.jailer_path}/dev/")
```

Comment:
Use `shutil.copy` or any other standard Python function. Calling bash processes is not needed here.